### ISSUE TITLE
url: traceback can become part of url

### ISSUE THREAD

**Issue by RhinosF1** [sopel-irc/sopel#2207]
### Description
The broken url given in tracebacks to IRC is not always correct

### Reproduction steps
1. Setup sopel
2. Connect to a server, join channel and set exceptions to go to IRC
3. Paste a broken url
4. Read exception

### Expected behavior
url not include traceback on the end

### Logs
```
08:56:42 <RhinosF1> 08:54:54 <MirahezeBot> metawiki * [[Special:Log/block]] reblock  * RhinosF1 *  changed block settings for [[User:45.14.71.0/24]] with an expiration time of 1 year (anonymous users only, account creation disabled, email disabled, cannot edit own talk page): VPN - See [[NOP]] (ProtonVPN per https://whois-
08:56:42 <MirahezeBot> [2021-11-06 08:56:41,711] sopel.modules.url    ERROR    - Unable to reach URL: https://whois-Traceback (most recent call last):  File "/srv/sopelbots/prodvenv/lib/python3.7/site-packages/urllib3/connection.py", line 175, in _new_conn    (self._dns_host, self.port), self.timeout, **extra_kw  File "/srv/sopelbots/prodvenv/lib/python3.7/site-packages/urllib3/util/connection.py", line 73, in create_connection    for res in socket.getaddr
08:56:44 <MirahezeBot> [2021-11-06 08:56:41,714] sopel.modules.url    WARNING  - No title found; ignoring URL: https://whois-
```

### Environment
- Sopel `.version`: 7.1.5
- Sopel installed via: pip
- Python version: 3.7.3
- Operating system: Debian 10
- IRCd `/version`: 09:01:29 solanum-1.0-dev(20211009-430833dc). molybdenum.libera.chat eIKMpS6 TS6ow
- Relevant plugins: url

### Notes
Should we tell users that their URLs are broken rather than an exception anyway?

**Comment by Exirel** [comment]
What was the exact URL expected?

**Comment by RhinosF1** [comment]
`08:56:42 <MirahezeBot> [2021-11-06 08:56:41,711] sopel.modules.url    ERROR    - Unable to reach URL: https://whois- Traceback (most recent call last):  File "/srv/sopelbots/prodvenv/lib/python3.7/site-packages/urllib3/connection.py", line 175, in _new_conn    (self._dns_host, self.port), self.timeout, **extra_kw  File "/srv/sopelbots/prodvenv/lib/python3.7/site-packages/urllib3/util/connection.py", line 73, in create_connection    for res in socket.getaddr
`

is what I'd have expected to be outputed

**Comment by Exirel** [comment]
You wrote:

> The broken **url** given in tracebacks to IRC is not always **correct**

As far as I can tell, the broken URL was `https://whois-`. What I see in the logs you provided:

> Unable to reach URL: https://whois-

Looks like it is the same URL. What is the problem?

**Comment by dgw** [comment]
> Looks like it is the same URL. What is the problem?

The missing space before "Traceback". That's literally all.

> Should we tell users that their URLs are broken rather than an exception anyway?

`url` is intended to fail silently if `auto_title()` can't fetch a title in a case like this, because there are many reasons why a title might not be found and auto-title is merely an opportunistic convenience. It's supposed to stay quiet unless it has something useful to say. (The explicit command `.title`, however, _will_ say if it had a problem fetching any titles, because commands are always expected to output _something_ describing the result of calling them.)

These failures are only sent to the log for informational purposes. Logging-to-IRC is a convenience only, and I wouldn't configure Sopel to output logs on a public IRC channel, anyway; that's too technical for general users and only useful for the bot owner(s). Who, I might add, can and should look at the actual log files if there's a real problem.

I think the wrong logging level is being used. They seem like INFO or DEBUG level messages, certainly not ERROR. (We could just call `LOGGER.info('whatever message', exc_info=True)`?)

**Comment by dgw** [comment]
@RhinosF1 Is this sufficiently fixed by #2250 that we can consider it closed?

**Comment by dgw** [comment]
Considering as fixed by #2250, given lack of contrary feedback from reporter.
